### PR TITLE
Add Spanish translation to intro -> how IS works

### PR DESCRIPTION
**Pull request by dicortazar**
Translation of [How does InnerSource work?](https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/introduction/03-how-works.asciidoc)

### CODE REVIEW

**Comment by rrrutledge** [comment]
Thanks for this pull request, Daniel!  I am on vacation now but will see when I get back next week if I can take a look.

File: introduction/es/03-how-works.asciidoc [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+En un entorno de InnerSource, si al equipo A no le facilitan dicha funcionalidad entonces habría enviado un pull request en su lugar.
+Esto significa que el equipo A implementa la funcionalidad directamente en el software del equipo B y envía un pull request para que sea revisado con los cambios pertinentes.
+El equipo B es entonces el encargado de revisar dicho código y aceptarlo cuando esté listo.
+
+En este ejemplo, el equipo A es el _Invitado_ y el equipo B es el equipo _Anfitrión_.
+Los términos de _Invitado_ y _Anfitrión_ se usan para referirse a una situación análoga a la de tener una visita en casa.
+En esta situación, la mayor parte de las personas quieren ser un buen anfitrión y para ello hay que asegurar que la casa está ordenada y las cosas guardadas en su sitio como anticipo a la visita de nuestros invitados. Los visitantes serán entonces bienvenidos en la puerta e invitados a pasar y podrán usar las diferentes herramientas e instalaciones que sean públicas.
+Hay, por supuesto, una serie de relas en la casa que se pide a los invitados que sigan.
```
Comment (rrrutledge) [inline introduction/es/03-how-works.asciidoc:12]:
```suggestion
Hay, por supuesto, una serie de reglas en la casa que se pide a los invitados que sigan.
```

File: introduction/es/03-how-works.asciidoc [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+De igual manera, los invitados quieren ser educados y mostrar respeto por la casa y por los anfitriones. Tienen cuidado con las cosas de la casa y siguen las reglas durante su estancia. Además esperan que quizá vuelvan a ser invitados de nuevo siempre y cuando hayan sido educados y corteses.
+Estos conceptos acerca de una visita en una casa son una metáfora de la actitud  y comportamientos que los equipos deben tener cuando actúan como anfitriones o como invitados a la hora de contribuir al código fuente de nuestro producto.
+
+Echemos un vistazo más en detalle del proceso de InnerSource.
+Para ilustrar esta explicación usaremos unos pocos roles que son clave en los equipos que actúan como invitados y anfitriones.
+En primer lugar, el https://innersourcecommons.org/resources/learningpath/product-owner/index[_Product Owner_] determina qué funcionalidad puede ser aceptada por el equipo anfitrion como una contribución.
```
Comment (rrrutledge) [inline introduction/es/03-how-works.asciidoc:18]: Does the term _Product Owner_ have a Spanish translation in the Agile world?

File: introduction/es/03-how-works.asciidoc [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+Esta opción funciona para el equipo invitado porque consiguen la funcionalidad requerida cuando la necesitaban sin tener que hacerse cargo del proceso de mantenimiento de la misma en el medio o largo plazo.
+Funciona para el equipo anfitrión porque son capaces de acomodar y gestionar más trabajo a la vez que siguen sirviendo a sus consumidores.
+Además funciona para la compañía en su totalidad porque las soluciones a problemas compartidos terminan en lugares por todos conocidos, compartido y mantenido de forma centralizada que cualquiera puede usar.
+Finalmente, se invierte más tiempo de ingeniería en producir código que resuelve problemas de la compañía en lugar de estar enfocado en problemas políticos, jerárquicos y de negociaciones entre equipos.
+
+=== Recursos adicionales
+
+* El patrón del  https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/master/patterns/2-structured/trusted-committer.md[Trusted Committer].
```
Comment (rrrutledge) [inline introduction/es/03-how-works.asciidoc:41]:
```suggestion
* El patrón del https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/master/patterns/2-structured/trusted-committer.md[Trusted Committer].
```

**Review by rrrutledge — CHANGES_REQUESTED** [review]
Thank you for this submission!  Sorry for the delay in review.  A few comments.

File: introduction/es/03-how-works.asciidoc (hunk 2 above)
Comment (dicortazar) [reply introduction/es/03-how-works.asciidoc:18]: Not in reality (at last as far as I know!). We use product owner or PO.

I'd say this works in Spain (not sure about how this is translated, if so, in other Spanish-speaking countries).

**Comment by dicortazar** [comment]
Thanks for the suggestions @rrrutledge , I think I've covered all of them.

**Review by rrrutledge — APPROVED** [review]
Looks great!

**Comment by rrrutledge** [comment]
Opened a [pull request](https://github.com/InnerSourceCommons/innersourcecommons.org/pull/116) to put it on the website.

**Pull request merged**